### PR TITLE
Place :extensible-sequences in *features*

### PR DESCRIPTION
**Pull request by easye**
Addresses <https://github.com/armedbear/abcl/issues/363>.

### CODE REVIEW

File: src/org/armedbear/lisp/extensible-sequences.lisp [hunk 1]
```diff
@@ -1036,4 +1036,5 @@ the sequence argument to be a proper sequence.
                   (tagbody
                      ,@forms))))))))))
 
-(provide "EXTENSIBLE-SEQUENCES")
\ No newline at end of file
+(provide "EXTENSIBLE-SEQUENCES")
+(push :extensible-sequences *features*)
```
Comment (phoe) [inline src/org/armedbear/lisp/extensible-sequences.lisp:1040]: You might want a `pushnew` instead.

**Comment by phoe** [comment]
There is the rationale from https://bugs.launchpad.net/sbcl/+bug/1909800 that is convincing to me:

> Adding :extensible-sequences now would mean all older SBCL will stop supporting it.

Perhaps it would be better for `trivial-features` to push this feature on its own, based on the presence of `SEQUENCE` package in ABCL? (See https://github.com/trivial-features/trivial-features/issues/17)

File: src/org/armedbear/lisp/extensible-sequences.lisp (hunk 1 above)
Comment (easye) [reply src/org/armedbear/lisp/extensible-sequences.lisp:1040]: Yes, `pushnew` would always be preferred.

**Comment by dkochmanski** [comment]
ping pong :)

**Comment by phoe** [comment]
I no longer know how to proceed in this situation because of the argumentation in https://bugs.launchpad.net/sbcl/+bug/1909800:

> Adding :extensible-sequences now would mean all older SBCL will stop supporting it.

**Comment by dkochmanski** [comment]
I see, thank you.

**Comment by easye** [comment]
I'm going to approve this pull request, under the rubric of it doesn't matter how many symbols one has in `cl:*features*` 'cuz it has no namespaces in the first place.

**Pull request merged**